netbeans.apache.org/src/content/download/index.asciidoc

```diff
@@ -38,6 +38,7 @@ for important requirements for download pages for Apache projects.
 === Apache NetBeans 9.0
 
 - link:nb90/[Apache NetBeans 9.0 Features]
+- link:nb90/nb90.html[Apache NetBeans 9.0], released on the XXX of July, 2018.
```

Comment:
I think the target date is August 1st but perhaps we finish in July?

Reply:
Yes, of course, that has to be changed before the release.

Reply:
need to make sure that mirrors have the artifacts available before announcing anything, but using 29 July, since that's the date of announcing the IPMC vote result tally makes sense to me